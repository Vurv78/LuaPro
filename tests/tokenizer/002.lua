while if elseif else do then repeat local function return goto end for in until break