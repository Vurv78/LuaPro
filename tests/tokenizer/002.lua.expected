local lupa = require("src.lib.mod")
local Token, Variant = lupa.Token, lupa.Token.Variant

return {
	Token.new(Variant.Keyword, "while"),
	Token.new(Variant.Keyword, "if"),
	Token.new(Variant.Keyword, "elseif"),
	Token.new(Variant.Keyword, "else"),
	Token.new(Variant.Keyword, "do"),
	Token.new(Variant.Keyword, "then"),
	Token.new(Variant.Keyword, "repeat"),
	Token.new(Variant.Keyword, "local"),
	Token.new(Variant.Keyword, "function"),
	Token.new(Variant.Keyword, "return"),
	Token.new(Variant.Keyword, "goto"),
	Token.new(Variant.Keyword, "end"),
	Token.new(Variant.Keyword, "for"),
	Token.new(Variant.Keyword, "in"),
	Token.new(Variant.Keyword, "until"),
	Token.new(Variant.Keyword, "break")
}